hs_core/models.py

```diff
@@ -256,7 +257,7 @@ def page_permissions_page_processor(request, page):
         elif user_privilege == PrivilegeCodes.VIEW:
             self_access_level = 'view'
 
-    owners = cm.raccess.owners.all()
+    owners = cm.raccess.owners.filter(is_active=True).all()
```

Comment:
`is_active=True` is a filter parameter in the `hs_core.models.resource.ResourceAccess` `owners` property.